tests/api-controller-acceptance-tests/apicontroller/fixture.go

```diff
@@ -3,42 +3,45 @@ package apicontroller
 import (
 	"fmt"
 
+	"time"
+
 	log "github.com/sirupsen/logrus"
 	appv1 "k8s.io/api/apps/v1"
 	corev1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/intstr"
 	"k8s.io/client-go/kubernetes"
-	"time"
 )
 
 const (
-	namespace                                      = "kyma-system"
+	namespace                                      = "sampleapp"
```

Comment:
As the namespace is now expected to exist before test is run I'd suggest passing its name as environmental variable, not hardcoding it here.